pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -4,23 +4,171 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+const (
+	PhaseRealmCreated = "realm_created"
+	PhaseRealmReady   = "ready"
+)
 
 // KeycloakRealmSpec defines the desired state of KeycloakRealm
 // +k8s:openapi-gen=true
 type KeycloakRealmSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+	BrowserRedirectorIdentityProvider string                `json:"browserRedirectorIdentityProvider,omitempty"`
```

Comment:
What does it do?

Reply:
Good question. I copied it from the old keycloak operator. But maybe we should just leave it out until we encounter a use case where it's needed.